flake.nix

```diff
@@ -1,44 +1,72 @@
 {
   description = "plutarch";
 
-  inputs.haskell-nix.url = "github:input-output-hk/haskell.nix";
-  inputs.nixpkgs.follows = "haskell-nix/nixpkgs-unstable";
-  inputs.nixpkgs-latest.url = "github:NixOS/nixpkgs?rev=a0a69be4b5ee63f1b5e75887a406e9194012b492";
-
-  inputs.hercules-ci-effects.url = "github:hercules-ci/hercules-ci-effects";
-
-  inputs.haskell-nix-extra-hackage.url = "github:mlabs-haskell/haskell-nix-extra-hackage?ref=separate-hackages";
-  inputs.haskell-nix-extra-hackage.inputs.haskell-nix.follows = "haskell-nix";
-  inputs.haskell-nix-extra-hackage.inputs.nixpkgs.follows = "nixpkgs";
-
-  inputs.iohk-nix.url = "github:input-output-hk/iohk-nix";
-  inputs.iohk-nix.flake = false;
-  # we use sphinxcontrib-haddock input
-  inputs.plutus.url = "github:input-output-hk/plutus";
-  # https://github.com/input-output-hk/cardano-prelude/pull/163
-  inputs.cardano-prelude.url = "github:mlabs-haskell/cardano-prelude?rev=713c7ae79a4d538fcd653c976a652913df1567b9";
-  inputs.cardano-prelude.flake = false;
-  inputs.cardano-base.url = "github:input-output-hk/cardano-base";
-  inputs.cardano-base.flake = false;
-  inputs.cardano-crypto.url = "github:input-output-hk/cardano-crypto?rev=07397f0e50da97eaa0575d93bee7ac4b2b2576ec";
-  inputs.cardano-crypto.flake = false;
-  inputs.haskell-language-server.url = "github:haskell/haskell-language-server";
-  inputs.haskell-language-server.flake = false;
-
-  inputs.secp256k1-haskell.url = "github:haskoin/secp256k1-haskell";
-  inputs.secp256k1-haskell.flake = false;
-
-  # https://github.com/protolude/protolude/pull/133#issuecomment-1112150422 RC not uploaded to hackage yet...
-  inputs.protolude.url = "github:protolude/protolude";
-  inputs.protolude.flake = false;
-
-  # 0.4.5 hasn't been published to Hackage...
-  inputs.flat.url = "github:Quid2/flat";
-  inputs.flat.flake = false;
-
-  inputs.emanote.url = "github:srid/emanote/master";
-
-  outputs = inputs@{ self, nixpkgs, nixpkgs-latest, iohk-nix, haskell-nix, hercules-ci-effects, haskell-nix-extra-hackage, ... }:
+  inputs = {
```

Comment:
?

Reply:
especially as it gets more. it's also less to type.

Reply:
You have more than double the lines in an already long file.